<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>758</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>894</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MD interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Parameter Selection</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pb_tunes">
              <property name="text">
               <string>Add Tunes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_chrom">
              <property name="text">
               <string>Add Chroma</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_oct">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Add Ocutpole</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_other">
              <property name="text">
               <string>Other:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_search">
              <property name="placeholderText">
               <string>LHCBEAM1/TRIM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_search">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="list_search">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_search_add">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_selected">
              <property name="text">
               <string>Selected:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="columnCount">
               <number>3</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bp">
              <property name="text">
               <string>Beam Process:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_bp"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Random Generation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_n_points">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Number of points:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_n_points">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pb_generate">
              <property name="text">
               <string>Generate Trims</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_ref">
              <property name="text">
               <string>Reference Trims</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_optimize">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Optimize Sequence</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_load">
              <property name="text">
               <string>Load Sequence</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_save">
              <property name="text">
               <string>Save Sequence</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Trim Sequence</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_waittime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Wait time (s): </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_waittime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_simulate">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Simulate</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_go">
              <property name="text">
               <string>Go !</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_revert">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Revert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_forward">
              <property name="text">
               <string>Forward</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_stop">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_initial">
              <property name="text">
               <string>Initial</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_outfile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Output file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_outfile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_desc">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_desc">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Lifetime Feed</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="layout_lifetime"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_clear_lt">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
